_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Istio Multicluster
+overview: Install Istio with multicluster support.
+
+order: 65
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions for the installation of Istio multicluster.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters.
+
+* The usage of an RFC1918 network, VPN, or other more advanced network techniques
+to meet the following requirements:
+
+    * Individual cluster Pod CIDR ranges and service CIDR ranges must be unique
+across the multicluster environment and may not overlap.
+
+    * All nodes' pod CIDR in every cluster must be routable to every other nodes'
+pod CIDR.
+
+    * All Kubernetes control plane API servers must be routable to each other.
+
+* Helm **2.7.2 or newer**.  The use of Tiller is optional.
+
+* If using Tiller, the ability to modify RBAC rules required to install Helm
```

Comment:
I feel we are missing a verb here... If using Tiller, the ability to modify RBAC rules required -> requires? to install Helm

Reply:
done.